arrow-fx/src/test/kotlin/arrow/fx/MVarTest.kt

```diff
@@ -260,9 +263,9 @@ class MVarTest : UnitSpec() {
           val channel = !mvar.empty<Int>()
           val (count, reads, writes) = testStackSequential(channel)
           !writes.fork()
-          !reads
-          !effect { reads shouldBe count }
-        }.equalUnderTheLaw(IO.unit, EQ())
+          val r = !reads
+          !effect { r shouldBe count }
```

Comment:
This seems like the only relevant fix?

Reply:
Yep, only this nit and the Int -> Long below :)